lib/src/data/generators/padding_generator.dart

```diff
@@ -0,0 +1,227 @@
+import 'package:code_builder/code_builder.dart';
+import 'package:dart_style/dart_style.dart';
+import 'package:figmage/src/data/generators/generator_util.dart';
+import 'package:figmage/src/domain/generators/theme_class_generator.dart';
+
+///{@template padding_generator}
+///A generator for a padding class.
+///{@endtemplate}
```

Comment:
Laut [effective dart](https://dart.dev/effective-dart/documentation) immer ein space nach den slashes 
```suggestion
/// {@template padding_generator}
/// A generator for a padding class.
/// {@endtemplate}
```